external/include/stress_tool/st_perf.h

```diff
@@ -180,6 +180,16 @@ void perf_set_keeprunning(int enable);
 /*
  * Expect?
  */
+
+#define ST_EXPECT_ASSERT(val, exp) 						\
```

Comment:
Thanks for modification.
However I changed the macro name ST_EXPECT_ASSERT to ST_ASSERT_EQ . So you don't need to add it. 
I'll raised patch PR https://github.com/Samsung/TizenRT/pull/5321

Reply:
thank you, changes done.
Please merge